system/ee/language/english/pro_variables_lang.php

```diff
@@ -1,17 +1,5 @@
 <?php
 
-if (! defined('BASEPATH')) {
```

Comment:
@intoeetive I'm guessing this was removed on accident?

Reply:
The whole section here.. not just this one line.

Reply:
No, it was removed to bring the files to same format that other `lang` files have